lib/manager/npm/update/locked-dependency/package-lock/index.ts

```diff
@@ -47,6 +47,12 @@ export async function updateLockedDependency(
       depName,
       currentVersion
     );
+    if (lockedDeps.some((dep) => dep.bundled)) {
```

Comment:
Shouldn't we just filter bundled and update not bundled?

Reply:
Scenario:
- the vulnerable version is found twice
- one is bundled and the other is unbundled
- per your suggestion, we filter out the bundled version and only remediate the unbundled
- we raise a PR saying we've remediated the vulnerability
- PR is merged.. but vulnerability (including alert) remains

I don't like this edge case, so prefer to just stop trying